plugins/listblock/plugin.js

```diff
@@ -83,8 +84,11 @@ CKEDITOR.plugins.add( 'listblock', {
 			},
 
 			proto: {
-				add: function( value, html, title ) {
-					var id = CKEDITOR.tools.getNextId();
+				add: function( value, html, title, language, editor ) {
+					var id = CKEDITOR.tools.getNextId(),
+						editorLanguage = editor.config.language ||
+							editor.config.defaultLanguage ||
+							editor.langCode;
```

Comment:
Do we need so many options? Couldn't we use just `editor.langCode`? It should contain the actual language of the editor.

Reply:
Fixed. I added `lang` instead of `language` in the editor config.

Reply:
Removed as we changed our approach.